src/composables/map/street-view.composable.ts

```diff
@@ -0,0 +1,255 @@
+    }
+  })
+
+  watch(locationInfo, loc => {
+    if (loc) {
+      if (isStreetviewActive.value && !panoPositionChanging.value) {
+        setLocation(loc)
+      }
+    } else {
+      svFeature.value = undefined
+      map.getViewport().style.cursor = ''
+    }
+  })
+
+  watch(isStreetviewActive, async act => {
+    if (act) {
+      await loadGoogleapis()
+      if (window.hasOwnProperty('google')) {
+        // @ts-ignore
+        googleService = window.google
+      }
+      // todo PIWIK
+      if (streetViewService === null) {
+        streetViewService = new googleService!.maps.StreetViewService()
+      }
+      if (panorama === null) {
+        panorama = new googleService!.maps.StreetViewPanorama(
+          document.getElementById('streetview-div'),
+          {
+            pov: {
+              heading: 0,
+              pitch: 0,
+              zoom: 1,
+            },
+            visible: false,
+            zoom: 1,
+          }
+        )
+      }
+      if (panoramaLinksListener === null) {
+        panoramaLinksListener = googleService?.maps.event.addListener(
+          panorama,
+          'links_changed',
+          () => handlePanoramaPositionChange(true)
+        )
+      }
+      if (panoramaPovListener === null) {
+        panoramaPovListener = googleService?.maps.event.addListener(
+          panorama,
+          'pov_changed',
+          () => handlePanoramaPositionChange(false)
+        )
+      }
+      setLocation(locationInfo.value)
+    } else {
+      if (panorama !== null) {
+        panorama.setVisible(false)
+      }
+      svFeature.value = undefined
+      if (panoramaPovListener) {
+        googleService?.maps.event.removeListener(panoramaPovListener)
+        panoramaPovListener = null
+      }
+      if (panoramaLinksListener) {
+        googleService?.maps.event.removeListener(panoramaLinksListener)
+        panoramaLinksListener = null
+      }
+    }
+  })
+
+  watch(svFeature, svf => {
+    svFeatureLayer.getSource()?.clear()
+    if (svf) {
+      svFeatureLayer.getSource()?.addFeature(svf.compass)
+      svf.directions.forEach((f: SvDirectionFeature) =>
+        svFeatureLayer.getSource()?.addFeature(f)
+      )
+      hidePointer.value = true
+      map.addInteraction(selectFeature)
+      map.on('pointermove', handleHover)
+    } else {
+      map.un('pointermove', handleHover)
+      map.removeInteraction(selectFeature)
+      hidePointer.value = false
+    }
+  })
+
+  function handleNavigate(evt: BaseEvent) {
+    if (evt.target.getFeatures().getLength() !== 0) {
+      const nextDirection = evt.target
+        .getFeatures()
+        .getArray()[0] as SvDirectionFeature
+      evt.target.getFeatures().clear()
+      panorama.setPano(nextDirection.pano)
+    }
+  }
+
+  function handleHover(event: MapBrowserEvent<PointerEvent>) {
+    if (isStreetviewActive.value) {
+      const hit = map.forEachFeatureAtPixel(
+        event.pixel,
+        function (feature /*, layer*/) {
```

Comment:
clean commnet

Reply:
because basically it is a 2 argument callback function but the linter complains about the second argument not being used, so this keeps them visible